setup.py

```diff
@@ -25,38 +29,64 @@ def package_files(directory):
     "../mlflow/temporary_db_migrations_for_pre_1_users/alembic.ini",
 ]
 
+requirements = [
+    "click>=7.0",
+    "cloudpickle",
+    "databricks-cli>=0.8.7",
+    "requests>=2.17.3",
+    "six>=1.10.0",
+    "python-dateutil",
+    "protobuf>=3.6.0",
+    "gitpython>=2.1.0",
+    "pyyaml",
+    "querystring_parser",
+    "simplejson",
+    "entrypoints",
+    "gorilla",
+    "pandas",
+    "numpy",
+    "sqlparse",
```

Comment:
I need to check on this. I remember search runs needed this. And I could not get the RestStore tests to run easily because the server code requires extra dependencies, so the FileStore was useful for testing at a minimum. We may be able to find a way of creating the server in a separate conda env on a specified port for builds